### ISSUE TITLE
WARNING: Mod Organizer 2 does NOT work with Fallout New Vegas

### ISSUE THREAD

**Issue by aitte2** [TanninOne/modorganizer#1126]
MO2 removed the ability to manage BSA loading and load order, which means no way to load "archive invalidation.bsa" first, etc. It is a known issue: http://forum.step-project.com/topic/9377-mod-organizer-2-discussion-and-feedback/

To get all New Vegas mods to work, I had to downgrade to MO 1.3.11.

**Comment by WildfireDarkstar** [comment]
Can't the archive invalidation BSA still be specified manually in FalloutPrefs.ini?

**Comment by aitte2** [comment]
True, it could be. But we would still get the problem that many mods need to be loaded in a special order. Like "first EVE, then Project Blabla, then Project Blabla for EVE compatibility fix BSA". So the old Mod Organizer is best for old games like Fallout 3 and New Vegas.
